spras/containers.py

```diff
@@ -239,6 +242,40 @@ def run_container_docker(container: str, command: List[str], volumes: List[Tuple
     # finally:
     return out
 
+# TODO any issue with creating a new client each time inside this function?
```

Comment:
Any insight on whether this comment affects the podman client? I'd imagine this approach means that workflow runs with high concurrency will spawn many clients that each need to establish a connection with the podman socket. It may be useful to adjust the comment to explain why we're worried about this.